docs/08-developer-guide/04-project/03-release-guide.md

```diff
@@ -4,6 +4,372 @@ sidebar_label: Release Manager Guide
+
+```bash title="Publish key (committer)"
+svn co https://dist.apache.org/repos/dist/release/ozone
+cd ozone
+export CODESIGNINGKEY=your_gpg_key_id
+gpg --list-sigs $CODESIGNINGKEY >> KEYS
+gpg --armor --export $CODESIGNINGKEY >> KEYS
+svn commit -m "ozone: adding key of XXXX to the KEYS"
+```
+
+In case you are only a committer and not a PMC member, you can add your key to the dev KEYS file and a PMC can move it to the final destination
+
+```bash title="Move key (PMC)"
+#use latest KEYS as the base
+svn rm https://dist.apache.org/repos/dist/dev/ozone/KEYS
+svn cp https://dist.apache.org/repos/dist/release/ozone/KEYS  https://dist.apache.org/repos/dist/dev/ozone/KEYS
+
+svn co https://dist.apache.org/repos/dist/dev/ozone
+cd ozone
+export CODESIGNINGKEY=your_gpg_key_id
+gpg --list-sigs $CODESIGNINGKEY >> KEYS
+gpg --armor --export $CODESIGNINGKEY >> KEYS
+svn commit -m "ozone: adding key of XXXX to the KEYS"
+```
+
+## Pre-Vote
+
+### Create a parent Jira for the release
+
+This provides visibility into the progress of the release for the community. Tasks mentioned in this guide like cherry-picking fixes on to the release branch, updating the ozone website, publishing the docker image, etc can be added as subtasks.
+
+### Bulk comment on Jiras targeting this release
+
+Issue a Jira query like [this](https://issues.apache.org/jira/browse/HDDS-4183?jql=project%20%3D%20HDDS%20AND%20resolution%20%3D%20Unresolved%20AND%20(cf%5B12310320%5D%20%3D%201.3.0%20OR%20fixVersion%20%3D%201.3.0)%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC), modified for the release number you are working with, to find all unresolved Jiras that have the target version field set to this release. Note that some people incorrectly use Fix Version as the Target Version, so Fix Version is included in this search. Use the following steps to issue a bulk update to these Jiras:
+
+ 1. In the top right corner, click `Tools` and under `Bulk Change` , select `all ... issues`.
+ 2. Click the top check box to select all the issues. Click `Next`.
+ 3. Select `Edit Issues` , then click `Next`.
+ 4. Select `Change Fix Version/s` , and in the drop down select `Clear field`.
+    :::note
+    This corrects unresolved issues which incorrectly set a fix version for this release.
+ 5. Select `Change Target Version/s`, and enter the version of the release after the one you are managing.
+ 6. Select `Change Comment`, and add a comment saying that you have moved the release field out, but if the issue is being actively worked on, is close to completion, and would like to be included in this release, to contact you by a given date, probably a week in the future.
+    :::note
+    Even though the action is called `Change Comment`, it actually adds a comment to the Jira and does not affect existing comments.
+ 7. Keep clicking through until the operation is started. It may take a while for Jira to finish the bulk update once it is started.
+
+### Branching
```

Comment:
Keeping the header names formatted as commands makes the mini-map read like an instruction manual. All other headings are in this form except this one.
```suggestion
### Create a Release Branch
```